code/datums/quirks/positive_quirks/cyborg_lover.dm

```diff
@@ -0,0 +1,36 @@
+/datum/quirk/cyborg_lover
+	name = "Cyborg Lover"
+	desc = "You find silicon life forms fascinating! You like inspecting and touching their hulls and robo-bodies, as well you like being touched by their manipulators."
+	icon = FA_ICON_ROBOT
+	value = 2
+	mob_trait = TRAIT_CYBORG_LOVER
+	gain_text = span_notice("You are fascinated by silicon life forms.")
+	lose_text = span_danger("Cyborgs and other silicons aren't cool anymore.")
+	medical_record_text = "Patient reports being attracted by silicon life forms."
```

Comment:
🤨

Reply:
maybe you mean fascinated? attracted _by_ doesn't fit right